source/scripting/Vehicle.hpp

```diff
@@ -311,12 +311,13 @@ namespace GTA
 	};
 	public enum class VehicleWindowTint
 	{
+		GrangerNone = -1,
```

Comment:
I know, but doesn't the Granger have tinted rear windows and clear front windows?